migrate/migrate.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+// Set-up
+use CommunityVoices\App\Website\Bootstrap;
+
+// Timezone
+// @config
+date_default_timezone_set('America/New_York');
+
+// Error settings
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+// Composer autoloading
+require __DIR__ . '/../vendor/autoload.php';
+require __DIR__ . '/../src/CommunityVoices/App/Website/db.php';
+
+if (count($argv) < 2) {
+    echo "You must input a script name as an argument.\n";
+} else {
+    require __DIR__ . "/scripts/{$argv[1]}.php";
+}
+shell_exec ('mysqldump -uroot -p --no-data community_voices > migrate/schema.sql'); // this doesn't work within docker
```

Comment:
Reasoning for adding this into the migrate script that has to run on each migration run?  I think it would be better to have a separate script for running this.

Reply:
What was causing this not to work within Docker?  I'm especially wondering because this script has to be run within Docker.

Reply:
@Sammidysam I am pretty sure that you wrote this file haha so I am not sure

Reply:
Yes, now that you mention it, I do see the merit in this line.  This ensures that we update our schema always to make sure that the file helps us in generating blank databases correctly.

Are you sure you didn't write the comment stating this does not work within Docker?  I feel like I wrote the file and you wrote this specific line.  I just wouldn't put a space before the `(` after `shell_exec`.

Reply:
I mostly ask that because I need to figure out what is wrong with the line.

Reply:
Fixed